rllib/algorithms/algorithm.py

```diff
@@ -1576,9 +1572,7 @@ def restore_workers(self, workers: WorkerSet) -> None:
                 remote_worker_ids=restored,
                 # Don't update the local_worker, b/c it's the one we are synching from.
                 local_worker=False,
-                timeout_seconds=self.config.worker_restore_timeout_s,
-                # Bring back actor after successful state syncing.
-                mark_healthy=True,
+                timeout_seconds=self.config.env_runner_restore_timeout_s,
```

Comment:
Does this timeout and restoration time in general influence in any way the sampling (blocking it or slowing down for example)? I ask b/c in this case the `sample_timeout_s` could lead to no samples returned by `synchronous_parallel_sample` and lead to errors.